web/vue/src/components/global/mixins/dataset.js

```diff
@@ -21,13 +21,16 @@ var mixin = {
 
         response.datasets.forEach(market => {
           market.ranges.forEach(range => {
+            let num = 1;
```

Comment:
great observation, I totally forgot about the second param of the forEach. Will modify it and resubmit